### DISCUSSION TITLE
[QUESTION] Add Target to Publish Dialog

### DISCUSSION THREAD

**Discussion by dtillman** [VsixCommunity/Community.VisualStudio.Toolkit#197] [Q&A]
@madskristensen : 
Do you have an example of how to add to the publish dialog in VS2019.  
I'd like to add another "Target" to the dialog.
Thank you

**Comment by reduckted** [comment]
I don't think you can. I just had a look into this, and the interfaces that you'd need to implement all seem to be internal. If you want to look further, the assembly that defines those interfaces is `Common7\IDE\Extensions\Microsoft\Web Tools Shared\Microsoft.WebTools.Shared.VS.dll`.

![image](https://user-images.githubusercontent.com/10321525/135803114-fedd08da-cb2c-4c03-9ee5-f064a4469f7b.png)
